Investigate and fix build error in `wasm-test-runner`

Currently our `wasm-test-runner` fails with 
```
error[E0599]: no method named `clone` found for struct `JsStatic` in the current scope
  --> /home/th4s/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-rayon-1.2.1/src/lib.rs:81:13
   |
81 |         URL.clone()
   |             ^^^^^ method not found in `JsStatic<JsString>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `wasm-bindgen-rayon` (lib) due to 1 previous error
```
To reproduce go into `wasm-test-runner` and do a `cargo clean && cargo update` then execute `./run.sh`.

--- a/crates/wasm/Cargo.toml
+++ b/crates/wasm/Cargo.toml
@@ -44,7 +44,7 @@ tracing = { workspace = true }
 tracing-subscriber = { workspace = true, features = ["time"] }
 tracing-web = { version = "0.1" }
 tsify-next = { version = "0.5", default-features = false, features = ["js"] }
-wasm-bindgen = { version = "0.2" }
+wasm-bindgen = { version = "=0.2.92" }
 wasm-bindgen-futures = { version = "0.4" }
 # Use the patched ws_stream_wasm to fix the issue https://github.com/najamelan/ws_stream_wasm/issues/12#issuecomment-1711902958
 ws_stream_wasm = { git = "https://github.com/tlsnotary/ws_stream_wasm", rev = "2ed12aad9f0236e5321f577672f309920b2aef51" }

